### PR TITLE
Added documentation for percentile_cont

### PR DESCRIPTION
**Pull request by Autophagy**
@chaudum please review

### CODE REVIEW

File: blackbox/docs/sql/aggregation.txt [hunk 1]
```diff
@@ -373,6 +373,36 @@ strings.  The advantage is that the ``arbitrary`` function does very little
 to no computation as for example ``max`` aggregation function would
 do.
 
+
+percentile_cont
+===============
+
+The ``percentile_cont`` aggregation function, when provided a percentage and a
+sorted set, will return a single value above/below which the given 
+percentage of values in the sorted set reside.
```
Comment (chaudum) [inline blackbox/docs/sql/aggregation.txt]: suggestion: try to make the sentence less complicated

make 2 sentences for single value and array value input

File: blackbox/docs/sql/aggregation.txt [hunk 2]
```diff
@@ -373,6 +373,36 @@ strings.  The advantage is that the ``arbitrary`` function does very little
 to no computation as for example ``max`` aggregation function would
 do.
 
+
+percentile_cont
```
Comment (chaudum) [inline blackbox/docs/sql/aggregation.txt]: also: what does `_cont` stand for - maybe add a short sentence what it means

**Comment by matriv** [comment]
Shouldn't we add documentation for the WITHIN GROUP or will it be in a separate PR?

**Comment by Autophagy** [comment]
Any better @chaudum ?

**Comment by chaudum** [comment]
@matriv agree, we should also document the options for WITHIN GROUP. 

however "WITHIN GROUP" is kind of a special case, since it is part of the function syntax (the whole function signature is `percentile_cont(...) GROUP WITHIN (ORDER BY ...)` - that's why I would put it directly next to `percentile_cont`. also postgres docs list the percentile function on the aggregate functions page. what do you think?

**Comment by matriv** [comment]
@chaudum Since as I can see this syntax is only relevant to aggregate functions (PG docs), I would keep that together with the percentile_cont() function as you suggest.

File: blackbox/docs/sql/aggregation.txt [hunk 3]
```diff
@@ -373,6 +373,59 @@ strings.  The advantage is that the ``arbitrary`` function does very little
...
+adjacent items if required. The function takes as input either a single fraction
+or an array of fractions. When supplied a single fraction, the function will 
+return a single value corresponding to the percentile of the specified fraction.
+When supplied an array of fractions, the function will return an array of values
+corresponding to the percentile of each fraction specified.
+
+within group
+------------
+
+The ``within group`` clause produces an ordered set for use in ordered-set
```
Comment (chaudum) [inline blackbox/docs/sql/aggregation.txt]: ~~produces ...~~ defines the column and order for use in ...

Comment (chaudum) [inline blackbox/docs/sql/aggregation.txt]: actually ... *produces* is ok

**Comment by chaudum** [comment]
lgtm. please squash and merge into feature branch

**Pull request merged**